slam-libraries/viam-orb-slam3/orbslam_server_v1.h

```diff
@@ -52,6 +55,10 @@ class SLAMServiceImpl final : public SLAMService::Service {
         ServerContext *context, const GetPointCloudMapRequest *request,
         GetPointCloudMapResponse *response) override;
 
+    ::grpc::Status GetInternalState(
```

Comment:
[nit] Can we be consistent with whether we are commenting the gRPC methods (either add comments to GetInternalState and GetPositionNew, or remove comment from GetPointCloudMap)?

Reply:
done